pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -676,7 +685,7 @@ func (o *MustGatherOptions) Run() error {
 	}
 	defer o.logTimestamp()
 
-	queue := workqueue.NewRateLimitingQueue(workqueue.DefaultControllerRateLimiter())
+	queue := workqueue.NewTypedRateLimitingQueue[*corev1.Pod](workqueue.DefaultTypedControllerRateLimiter[*corev1.Pod]())
```

Comment:
That would be really nice to have this change in a separate commit. Just a note for the future PRs: Mixture of different kind of changes in a single commit is difficult to track for reviewer.